archive/multiplexer.go

```diff
@@ -57,7 +57,7 @@ func NewMultiplexer(out io.WriteCloser, shutdownInputs notifier) *Multiplexer {
 		},
 	}
 	mux.selectCases = []reflect.SelectCase{
-		reflect.SelectCase{
```

Comment:
Besides, we don't usually touch the unrelated code if there are no bugs or lint error.

Reply:
I just ran the `golangci` linter over the whole repo, as specified in our internal Wiki. It put in a lot more changes than I'd initially anticipated, but I figured I'd leave the changes in here in favor of what the Wiki says.

If the Wiki isn't up-to-date or if something's failing, then it'd be simple to drop this commit and apply the desired linting changes. I just like to make sure I run the linter that the CI uses before I push so that I'm not caught off guard by any CI failures.

Reply:
Can you point to which wiki page you refer to?

Reply:
Sure! I'll send it privately, because I'm not sure if it's a public document.

Reply:
You can paste the link here, and anybody who isn't supposed to be able to access the page won't be able to. It's better to keep these discussions in-band as much as possible so that other readers now and in the future can follow along

Reply:
Will keep that in mind in the future.

To be thorough, I was considering the [Tools and Mongomirror Orientation doc](https://wiki.corp.mongodb.com/x/xJmpBg), specifically the "Formatting" section. I'm using v1.14.4 of the Go toolchain and v1.27.0 of the `golangci-lint` script. @mattChiaravalloti confirmed that, even though nobody has likely run this version of `golangci-lint` over the repo in a while, it should be fine as long as the Evergreen tasks work out.